### DISCUSSION TITLE
Seed data (food, ingredient, unit, else)?

### DISCUSSION THREAD

**Discussion by hcientist** [TandoorRecipes/recipes#294] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not a problem, per se

**Describe the solution you'd like**
I would like for a fresh install to be more immediately useful.

**Describe alternatives you've considered**
1. Perhaps the developer or other experienced users would be willing to share their food, ingredient, and/or unit tables/data? Just as a link in this repo's wiki?
2. Perhaps such data could be committed to the repo as [initial Django data](https://docs.djangoproject.com/en/3.1/howto/initial-data/)?

**Additional context**
It seems that at least in a locale-specific way, there may be common staples that users might enjoy not entering manually.

**Comment by vabene1111** [comment]
As you have already mentioned, since everything is locale specific i did not provide any inital data. I guess people could post .json files with their data `manage.py dumpdata cookbook.food` or others and upload them to a discussion. Others could then import them using the `loaddata` command.

**Reply by V1d1o7** [reply]
I think vabene put it best, not only locale spefic, but user too. For example in my ingredients I have:
"Onion"
"Onion, Diced"
"Onion, Minced"
"Onion, Chopped"
Same with garlic, and some others. I do like the Discussion idea though. That said, I would bet low on the priority list, but not too hard to implement. What would be some "staples", @hcientist? The other thing would be in America it is "Half and Half' while UK and other places it is "Single Cream", so maybe those that translate could provide any data that could be useful in a locale specific way.

**Reply by reibuehl** [reply]
I am a new user and I also had the same thought as @hcientist . At least for the units, a starter config would be nice. Maybe with an optional Myanmar/Liberia/USA version ;-)
